backend/src/test/java/com/phaete/backend/forage/controller/ForageMapItemControllerIntegrationTest.java

```diff
@@ -0,0 +1,413 @@
+							]
+						},
+						"customMarker": {
+							"id": "1",
+							"position": [
+								51.0,
+								10.0
+							],
+							"iconUrl": "test-icon",
+							"iconSize": [
+								64,
+								64
+							],
+							"iconAnchor": [
+								32,
+								64
+							],
+							"popupAnchor": [
+								0,
+								64
+							],
+							"popupText": "Test popup"
+						},
+						"quantity": "ABUNDANT",
+						"quality": "EXCELLENT",
+						"dateFound": "never",
+						"notes": "test"
+					}
+				"""));
+	}
+
+	@Test
+	void findAllForageMapItems_returnEmptyList_onEmpty_DB() throws Exception {
+		mockMvc.perform(MockMvcRequestBuilders.get("/api/forageMapItems"))
+				.andExpect(status().isOk())
+				.andExpect(content().json("[]"));
+	}
+
+	@Test
+	void findAllForageMapItems_returnsForageMapItems_onSuccess() throws Exception {
+		forageWikiItemRepository.save(
+				new ForageWikiItem(
+						"1",
+						"Apple Tree",
+						ForageCategory.FRUIT,
+						ForageSource.TREE,
+						"Apple Tree",
+						ForageSeason.FALL,
+						List.of("url-to-apple-tree-image")
+				)
+		);
+		customMarkerRepository.save(
+				new CustomMarker(
+						"1",
+						new double[] {51.0, 10.0},
+						"test-icon",
+						new int[] {64, 64},
+						new int[] {32, 64},
+						new int[] {0, 64},
+						"Test popup"
+				)
+		);
+		forageMapItemRepository.save(
+				new ForageMapItem(
+						"1",
+						new ForageWikiItem(
+								"1",
+								"Apple Tree",
+								ForageCategory.FRUIT,
+								ForageSource.TREE,
+								"Apple Tree",
+								ForageSeason.FALL,
+								List.of("url-to-apple-tree-image")
+						),
+						new CustomMarker(
+								"1",
+								new double[] {51.0, 10.0},
+								"test-icon",
+								new int[] {64, 64},
+								new int[] {32, 64},
+								new int[] {0, 64},
+								"Test popup"
+						),
+						ForageQuantity.ABUNDANT,
+						ForageQuality.EXCELLENT,
+						"never",
+						"test"
+				)
+		);
+
+		mockMvc.perform(MockMvcRequestBuilders.get("/api/forageMapItems"))
+				.andExpect(status().isOk())
+				.andExpect(content().json("""
+					[
+						{
+							"forageWikiItem": {
+								"id": "1",
+								"name": "Apple Tree",
+								"category": "FRUIT",
+								"source": "TREE",
+								"description": "Apple Tree",
+								"season": "FALL",
+								"imageURLs": [
+								  "url-to-apple-tree-image"
+								]
+							},
+							"customMarker": {
+								"id": "1",
+								"position": [
+									51.0,
+									10.0
+								],
+								"iconUrl": "test-icon",
+								"iconSize": [
+									64, 64
+								],
+								"iconAnchor": [
+									32, 64
+								],
+								"popupAnchor": [
+									0, 64
+								],
+								"popupText": "Test popup"
+							},
+							"quantity": "ABUNDANT",
+							"quality": "EXCELLENT",
+							"dateFound": "never",
+							"notes": "test"
+						}
+					]
+				"""));
+	}
+
+	@Test
+	void findAllForageMapItems_returnsForageMapItems_withoutInvalidCustomMarker() throws Exception {
```

Comment:
I don't understand what the tested method is expected to do in this test case.
Renaming might help.